__main__.py

```diff
@@ -41,26 +41,30 @@ async def get_prefix(bot, message):
 #
 async def _init_async():
     _pool = await aiomysql.create_pool(
-        host=mysql_host,
-        port=3306,
-        db=mysql_db,
-        user=mysql_user,
-        password=mysql_password,
-        autocommit=True,
-        loop=bot.loop
+        host       = mysql_host,
+        port       = 3306,
+        db         = mysql_db,
+        user       = mysql_user,
+        password   = mysql_password,
+        autocommit = True,
+        loop       = bot.loop
```

Comment:
This is not what I meant 🤦 but I guess it's fine.

Reply:
i tried